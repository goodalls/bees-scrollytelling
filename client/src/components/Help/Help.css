.Help {
  min-height: 100vh;
  color: inherit; }
  .Help h1 {
    font-size: 5em;
    margin: 20px 0; }
  .Help h2 {
    font-size: 3em;
    font-weight: bold; }
  .Help p {
    font-size: 2em; }
  .Help .flowers {
    position: relative;
    display: flex;
    height: 200px;
    width: 100%; }
  .Help .bee-images {
    height: 257px;
    width: 390px;
    background-repeat: no-repeat;
    background-size: contain;
    background-position: center; }
  .Help .bee-home {
    background-image: url("../../assets/bee-home1.png"); }
