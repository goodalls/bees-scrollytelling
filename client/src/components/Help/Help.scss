.Help {
  min-height: 100vh;
  color: inherit;

  & h1 {
    font-size: 5em;
    margin: 20px 0;
  }

  & h2 {
    font-size: 3em;
    font-weight: bold;
  }

  & p {
    font-size: 2em;
  }

  & .flowers {
    position: relative;
    display: flex;
    height: 200px;
    width: 100%;
  }

  & .bee-images {
    height: 257px;
    width: 390px;
    background-repeat: no-repeat;
    background-size: contain;
    background-position: center;
  }

  & .bee-home {
    background-image: url('../../assets/bee-home1.png')
  }
}