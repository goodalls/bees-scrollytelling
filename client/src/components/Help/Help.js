import React from 'react';
import './Help.css';
import flower1 from '../../assets/hypericum.svg';
import flower2 from '../../assets/iris.svg';
import flower3 from '../../assets/knapweed.svg';
import flower4 from '../../assets/mimosa.svg';
import flower5 from '../../assets/pansy.svg';
import flower6 from '../../assets/poppy.svg';
import flower7 from '../../assets/carnation.svg';

const Help = () => {
  return (
    <div className="Help">
      <h1>Ways you can help</h1>
      <h2>Buy organic and local</h2>
      <h2>Grow pollinator gardens</h2>
      <p>Be sure to ask if the plants you buy were grown with neonicitinoides</p>
      <div className='flowers'>
        <img className='flowers' src={flower1} />
        <img className='flowers' src={flower2} />
        <img className='flowers' src={flower3} />
        <img className='flowers' src={flower4} />
        <img className='flowers' src={flower5} />
        <img className='flowers' src={flower6} />
        <img className='flowers' src={flower7} />
      </div>
      <h2>Make bee homes</h2>
      <div className="bee-images bee-home">
      </div>
      <h2>Support bee friendly organizations and legislation</h2>
    </div>
  );
};

export default Help;