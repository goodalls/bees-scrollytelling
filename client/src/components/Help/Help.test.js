import React from 'react';
import Help from './Help';
import { shallow } from 'enzyme';

describe('Help', () => {
  it('matches the Snapshot', () => {
    const wrapper = shallow(<Help />);
    expect(wrapper).toMatchSnapshot();
  });
});