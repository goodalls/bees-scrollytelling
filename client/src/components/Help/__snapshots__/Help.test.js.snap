// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help matches the Snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Help />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Ways you can help
</h1>,
        <h2>
          Buy organic and local
</h2>,
        <h2>
          Grow pollinator gardens
</h2>,
        <p>
          Be sure to ask if the plants you buy were grown with neonicitinoides
</p>,
        <div
          className="flowers"
>
          <img
                    className="flowers"
                    src="hypericum.svg"
          />
          <img
                    className="flowers"
                    src="iris.svg"
          />
          <img
                    className="flowers"
                    src="knapweed.svg"
          />
          <img
                    className="flowers"
                    src="mimosa.svg"
          />
          <img
                    className="flowers"
                    src="pansy.svg"
          />
          <img
                    className="flowers"
                    src="poppy.svg"
          />
          <img
                    className="flowers"
                    src="carnation.svg"
          />
</div>,
        <h2>
          Make bee homes
</h2>,
        <div
          className="bee-images bee-home"
/>,
        <h2>
          Support bee friendly organizations and legislation
</h2>,
      ],
      "className": "Help",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Ways you can help",
        },
        "ref": null,
        "rendered": "Ways you can help",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Buy organic and local",
        },
        "ref": null,
        "rendered": "Buy organic and local",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Grow pollinator gardens",
        },
        "ref": null,
        "rendered": "Grow pollinator gardens",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Be sure to ask if the plants you buy were grown with neonicitinoides",
        },
        "ref": null,
        "rendered": "Be sure to ask if the plants you buy were grown with neonicitinoides",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              className="flowers"
              src="hypericum.svg"
/>,
            <img
              className="flowers"
              src="iris.svg"
/>,
            <img
              className="flowers"
              src="knapweed.svg"
/>,
            <img
              className="flowers"
              src="mimosa.svg"
/>,
            <img
              className="flowers"
              src="pansy.svg"
/>,
            <img
              className="flowers"
              src="poppy.svg"
/>,
            <img
              className="flowers"
              src="carnation.svg"
/>,
          ],
          "className": "flowers",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "flowers",
              "src": "hypericum.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "flowers",
              "src": "iris.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "flowers",
              "src": "knapweed.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "flowers",
              "src": "mimosa.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "flowers",
              "src": "pansy.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "flowers",
              "src": "poppy.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "flowers",
              "src": "carnation.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Make bee homes",
        },
        "ref": null,
        "rendered": "Make bee homes",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "bee-images bee-home",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Support bee friendly organizations and legislation",
        },
        "ref": null,
        "rendered": "Support bee friendly organizations and legislation",
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Ways you can help
</h1>,
          <h2>
            Buy organic and local
</h2>,
          <h2>
            Grow pollinator gardens
</h2>,
          <p>
            Be sure to ask if the plants you buy were grown with neonicitinoides
</p>,
          <div
            className="flowers"
>
            <img
                        className="flowers"
                        src="hypericum.svg"
            />
            <img
                        className="flowers"
                        src="iris.svg"
            />
            <img
                        className="flowers"
                        src="knapweed.svg"
            />
            <img
                        className="flowers"
                        src="mimosa.svg"
            />
            <img
                        className="flowers"
                        src="pansy.svg"
            />
            <img
                        className="flowers"
                        src="poppy.svg"
            />
            <img
                        className="flowers"
                        src="carnation.svg"
            />
</div>,
          <h2>
            Make bee homes
</h2>,
          <div
            className="bee-images bee-home"
/>,
          <h2>
            Support bee friendly organizations and legislation
</h2>,
        ],
        "className": "Help",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Ways you can help",
          },
          "ref": null,
          "rendered": "Ways you can help",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Buy organic and local",
          },
          "ref": null,
          "rendered": "Buy organic and local",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Grow pollinator gardens",
          },
          "ref": null,
          "rendered": "Grow pollinator gardens",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Be sure to ask if the plants you buy were grown with neonicitinoides",
          },
          "ref": null,
          "rendered": "Be sure to ask if the plants you buy were grown with neonicitinoides",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                className="flowers"
                src="hypericum.svg"
/>,
              <img
                className="flowers"
                src="iris.svg"
/>,
              <img
                className="flowers"
                src="knapweed.svg"
/>,
              <img
                className="flowers"
                src="mimosa.svg"
/>,
              <img
                className="flowers"
                src="pansy.svg"
/>,
              <img
                className="flowers"
                src="poppy.svg"
/>,
              <img
                className="flowers"
                src="carnation.svg"
/>,
            ],
            "className": "flowers",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "flowers",
                "src": "hypericum.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "flowers",
                "src": "iris.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "flowers",
                "src": "knapweed.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "flowers",
                "src": "mimosa.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "flowers",
                "src": "pansy.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "flowers",
                "src": "poppy.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "flowers",
                "src": "carnation.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Make bee homes",
          },
          "ref": null,
          "rendered": "Make bee homes",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "bee-images bee-home",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Support bee friendly organizations and legislation",
          },
          "ref": null,
          "rendered": "Support bee friendly organizations and legislation",
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
